apix-interface/src/main/java/eu/xenit/apix/search/SearchQueryConsistency.java

```diff
@@ -5,5 +5,6 @@
  */
 public enum SearchQueryConsistency {
     EVENTUAL,
-    TRANSACTIONAL
```

Comment:
You add TRANSACTIONAL_IF_POSSIBLE, but where does the processing of this new value happen? I see no code added for it.

Reply:
Kenneth explain IRL